samples/entity-framework-core/SqlPersistence_6/Endpoint.SqlPersistence/Endpoint.SqlPersistence.csproj

```diff
@@ -1,6 +1,6 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1;netcoreapp2.1;net48</TargetFrameworks>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
I get why you're taking out net48 in an EFCore sample, odd that it ever worked. Is there a reason not to allow netcoreapp2.1?